.github/workflows/custom_docker_builds.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           - docker-image: ./images/cache-indexer
             image-tags: ghcr.io/spack/cache-indexer:0.0.3
           - docker-image: ./analytics
-            image-tags: ghcr.io/spack/django:0.0.3
+            image-tags: ghcr.io/spack/django:0.0.4
```

Comment:
This needs to be bumped in https://github.com/spack/spack-infrastructure/blob/main/k8s/production/custom/webhook-handler/deployments.yaml as well right?

Reply:
😩